_data/simple-icons.json

```
@@ -2814,6 +2814,11 @@
             "hex": "EF4223",
             "source": "https://www.codeigniter.com/help/legal"
         },
+        {
+            "title": "Codeium",
+            "hex": "09B6A2",
+            "source": "https://codeium.com/"
```

```suggestion
            "source": "https://codeium.com"
```
Remove trailing slash